File: eNote/eNote.Model/DTOs/Instrumenti.cs
namespace eNote.Model.DTOs
{
    public class Instrumenti
    {
        public int Id { get; set; }
        public string Model { get; set; } = null!;
        public string Proizvodjac { get; set; } = null!;
        public string? Opis { get; set; } 
        public string VrstaInstrumenta { get; set; } = null!;
        public string MusicShop { get; set; } = null!;
        public byte[]? Slika { get; set; }
        public bool Dostupan { get; set; }
    }
}

File: eNote/UI/enote_desktop/lib/models/instrumenti.dart
import 'package:json_annotation/json_annotation.dart';

part 'instrumenti.g.dart';

@JsonSerializable()
class Instrumenti {
  int? id;
  String? model;
  String? proizvodjac;
  String? opis;
  String? slika;
  bool? dostupan;
  int? vrstaInstrumentaId;
  String? vrstaInstrumenta;
  int? musicShopId;
  String? musicShop;

  Instrumenti();

  factory Instrumenti.fromJson(Map<String, dynamic> json) =>
      _$InstrumentiFromJson(json);

  Map<String, dynamic> toJson() => _$InstrumentiToJson(this);
}
